### PR TITLE
Fix #2266: RTL Support in Admin Pin Verification

### LINKED ISSUE #2266: [RTL] High-fi AdminPinVerification [BLOCKED]
`AdminPinVerification` needs RTL support as shown in below screen.

PC AP (Empty) – RTL

Make sure that this works correctly in mobile (landscape + portrait) and tablet (landscape + portrait).

**Do not focus on keyboard, we are using native android keyboard.**

## References
Mobile Portrait: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/grid/
Mobile Landscape: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/grid/
Tablet: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/grid/
Wiki: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by RaghavAwasthi**
## Explanation

  The PR fixes #2266  by adding `locale ` based RTL Support for users.
  The PR  also adds the .idea folder to `.gitignore` so that android studio files get auto ignored by default  for all the contributors 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

## Description
resized

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,7 +10,7 @@
 /.idea/modules.xml
 /.idea/navEditor.xml
 /.idea/vcs.xml
-/.idea/workspace.xml
+/.idea
```
Comment (FareesHussain) [inline .gitignore]: This has a seperate issue for this #1734 and it is blocked on  `XML linter` so please revert this

File: app/src/main/res/layout-sw600dp-land/pin_password_activity.xml [hunk 2]
```diff
@@ -15,6 +15,8 @@
 
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
+    android:layoutDirection="locale"
+
```
Comment (FareesHussain) [inline app/src/main/res/layout-sw600dp-land/pin_password_activity.xml]: Remove this extra line

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@RaghavAwasthi Need reformatting. I suggest to set up a macro so it will be handy all the time, and it also works with XML files.
Check out wiki here - https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros

please check - https://github.com/oppia/oppia-android/wiki/RTL-Guidelines
Add screenshots in description
Also, I've added few comments please check

**Comment by FareesHussain** [comment]
@RaghavAwasthi Add screenshots in the description too

**Comment by RaghavAwasthi** [comment]
> @RaghavAwasthi Add screenshots in the description too

Done

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@RaghavAwasthi Screenshot should look like this add the same for all the layouts
image

**Comment by FareesHussain** [comment]
@RaghavAwasthi In RTL the input must be from **Right to Left** but for the `PinView` it is **Left to Right** please fix that too

**Comment by FareesHussain** [comment]
Sorry I forgot to mention it please update the title to :
`Fix #2266: RTL Support in Admin Pin Verification`

**Comment by anandwana001** [comment]
@RaghavAwasthi any update on this?

**Comment by RaghavAwasthi** [comment]
@anandwana001  I was quite ill, hence was not able to complete it. Will update in a day or so. Thanks

**Comment by Luffy18346** [comment]
@RaghavAwasthi any update on this?

**Comment by rt4914** [comment]
Closing this PR because of no activity.

**Pull request closed without merging**